client-v2/src/bundles/frontsUIBundle.ts

```diff
@@ -1,7 +1,11 @@
 import without from 'lodash/without';
+import compact from 'lodash/compact';
 import {
   Action,
+  EditorShowOpenFrontsMenu as EditorOpenCurrentFrontsMenu,
```

Comment:
any reason to not export this as `EditorOpenCurrentFrontsMenu` rather than alias on import?

Reply:
ah, a hangover from a VSCode F2 refactor, thanks for spotting 👍